packages/integration_test/lib/integration_test.dart

```diff
@@ -2,6 +2,20 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+/// Tool for self-driving testing of Flutter code on devices and emulators.
+///
+/// It adapts `flutter_test` results into a format that is compatible with
+/// `flutter drive` and native Android instrumentation testing.
+///
+/// To use, follow the [*Integration testing* guide](https://flutter.dev/docs/testing/integration-tests).
+///
+/// See also:
+///
+///  * [Usage section in the README](https://github.com/flutter/flutter/tree/master/packages/integration_test#usage),
+///    which fully details how to use the package, how to perform Android and
+///    iOS device testing.
+library integration_test;
```

Comment:
I am not sure if adding the library keyword here is necessary.

For my understanding `library` is used when generating documentation to feature a module as a library, but it already works like that for this module without it: It already appears in the documentation as library (see screenshot from https://api.flutter.dev/flutter/package-integration_test_integration_test/package-integration_test_integration_test-library.html)

![image](https://user-images.githubusercontent.com/2494376/144459392-b7faed11-38ee-4c45-850e-cfac500c6ac8.png)